Sources/TootSDK/TootClient/TootClient+TimeLine.swift

```diff
@@ -41,12 +41,13 @@ public extension TootClient {
     /// - Parameters:
     ///   - pageInfo: a PageInfo struct that tells the API how to page the response, typically with a minId set of the highest id you last saw
     ///   - limit: Maximum number of results to return (defaults to 20 on Mastodon with a max of 40)
+    ///   - onlyMedia: Return only statuses with media attachments
     /// - Returns: a PagedResult containing the posts retrieved
-    func getLocalTimeline(_ pageInfo: PagedInfo? = nil, limit: Int? = nil) async throws -> PagedResult<[Post]> {
+    func getLocalTimeline(_ pageInfo: PagedInfo? = nil, limit: Int? = nil, onlyMedia: Bool? = nil) async throws -> PagedResult<[Post]> {
```

Comment:
Even though it's a single, field, let's move this to a `LocalTimelineQuery` struct as a first optional parameter:
 ```suggestion
    func getLocalTimeline(_ query: LocalTimelineQuery? = nil, _ pageInfo: PagedInfo? = nil, limit: Int? = nil) async throws -> PagedResult<[Post]> {
```